rust/agents/relayer/src/msg/processor.rs

```diff
@@ -178,7 +169,12 @@ impl MessageProcessor {
             }
         }
         let checkpoint = ckpt.unwrap();
-        assert!(checkpoint.checkpoint.index >= self.message_leaf_index);
+        assert!(checkpoint.checkpoint.index >= self.message_nonce);
+        info!(
```

Comment:
wondering if this was intentional to keep as info- may be better suited as a debug

Reply:
Ah no it wasn't, removed.